src/commands/deploy/title-escrow.ts

```diff
@@ -0,0 +1,71 @@
+import { Argv } from "yargs";
+import { error, success, info } from "signale";
+import { getLogger } from "../../logger";
+import { deployTitleEscrow } from "../../implementations/deploy/title-escrow";
+import { DeployTitleEscrowCommand } from "./deploy.types";
+
+const { trace } = getLogger("deploy:title-escrow");
+
+export const command =
+  "title-escrow <token-registry-address> <beneficiary> <holder> <titleEscrowFactoryAddress> [options]";
+
+export const describe = "Deploys a title escrow on the blockchain";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
```

Comment:
I'm wondering if we should use only option rather than positional

Reply:
Hmm i think when they are compulsory, it's better to use positional, if not you need to add flags everywhere

Reply:
at least flags make it explicit what you put in what fields, while positional ... must remember. also with positional if you had things or change slightly then you break things. 

FOr instance it happens with wrap which expected a directory as output at the beginning and for which it's not the case anymore ... I had to bread the API to support a different behavior.

can think of the same problem in js : fn(a,b,c,d) vs fn({a,b,c,d})

Reply:
I'm running out of alphabets

Reply:
![image](https://user-images.githubusercontent.com/7406870/83724708-52debb80-a673-11ea-8cce-506f9ceab106.png)

done😂

Reply:
lol, alias are not required for option just saying xD, I dont approve for you to read, but I'm fin ewith the changes